packages/dialog/test/draggable-resizable.test.js

```diff
@@ -74,10 +74,10 @@ describe('helper methods', () => {
   it('should set bounds correctly', () => {
     const overlayBounds = dialog1.$.overlay.getBounds();
     dialog1.$.overlay.setBounds(overlayBounds);
-    expect(Math.floor(overlayPart.style.top)).to.be.eql(Math.floor(overlayBounds.top + 'px'));
-    expect(Math.floor(overlayPart.style.left)).to.be.eql(Math.floor(overlayBounds.left + 'px'));
-    expect(Math.floor(overlayPart.style.width)).to.be.eql(Math.floor(overlayBounds.width + 'px'));
-    expect(Math.floor(overlayPart.style.height)).to.be.eql(Math.floor(overlayBounds.height + 'px'));
+    expect(Math.floor(overlayPart.style.top)).to.be.eql(Math.floor(`${overlayBounds.top}px`));
```

Comment:
This test appears to be wrong because `Math.floor()` would return `NaN` in all cases due to `px` unit usage.

Let's update it to not check for `px` but instead compare numbers e.g. like this:

```js
expect(parseFloat(overlayPart.style.top)).to.be.closeTo(overlayBounds.top, 1);
```

Reply:
Done.